### PR TITLE
 Add DataTransferManger interop APIs

### PR DESCRIPTION
**Pull request by Gaoyifei1011**
Add DataTransferManger interop APIs
Update some interop api supported version(Because some of the minimum versions required for WinRT interface interoperability apis do not correspond to the official documentation)

### CODE REVIEW

**Comment by Gaoyifei1011** [comment]
> @Gaoyifei1011 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: src/cswinrt/strings/ComInteropHelpers.cs [hunk 1]
```diff
@@ -127,7 +139,7 @@ namespace Windows.ApplicationModel.DataTransfer.DragDrop.Core
 {
 #if UAC_VERSION_1
 #if NET
-    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.22000.0")]
```
Comment (dongle-the-gadget) [inline src/cswinrt/strings/ComInteropHelpers.cs:142]: Why are these lines changed?

Comment (Gaoyifei1011) [reply src/cswinrt/strings/ComInteropHelpers.cs:142]: > Why are these lines changed?

Screenshot_2024-03-07-18-51-03-312_mark via

Because I found that some interfaces have a minimum version limit, referring to the Microsoft documentation, if the version is still 10240, it will cause misuse

Comment (Gaoyifei1011) [reply src/cswinrt/strings/ComInteropHelpers.cs:142]: and some api contract is update to 19（26072），so add uac_version to 19
Screenshot_2024-03-07-19-03-18-811_mark via

Comment (dongle-the-gadget) [reply src/cswinrt/strings/ComInteropHelpers.cs:142]: I think changes that are unrelated to the scope of the PR (in this case, `DataTransferManager`) should be avoided. You can probably submit another PR.

Comment (Gaoyifei1011) [reply src/cswinrt/strings/ComInteropHelpers.cs:142]: > or

Uh, all right. Then I will close this PR and write a new PR
